<launch>

  <node pkg="laser_assembler" type="laser_scan_assembler" output="screen"  name="laser_scan_assembler">
    <remap from="scan" to="dummy_scan"/>
    <param name="tf_cache_time_secs" type="double" value="10.0" />
    <param name="max_scans" type="int" value="400" />
    <param name="ignore_laser_skew" type="bool" value="true" />
    <param name="fixed_frame" type="string" value="dummy_base_link" />
  </node>

  <node pkg="pr2_laser_snapshotter" type="pr2_laser_snapshotter" output="screen" name="snapshotter">
    <remap from="laser_scanner_signal" to="dummy_signal"/>
    <remap from="build_cloud" to="build_cloud" />
    <remap from="full_cloud" to="dummy_cloud" />
  </node>

  <node pkg="pr2_laser_snapshotter" name="dummy_scan_producer" type="dummy_scan_producer" output="screen"/>

  <test test-name="test_laser_assembler" pkg="pr2_laser_snapshotter" type="test_assembler" />

</launch>
